lib/bolt_ext/schemas/ssh-run_task.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "SSH run_task request",
+  "description": "POST ssh/run_task request schema",
+  "type": "object",
+  "properties": {
+    "target": {
+      "type": "object",
+      "description": "Target information to run task on",
+      "properties": {
+        "hostname": {
+          "type": "string",
+          "description": "Target identifier"
+        },
+        "user": {
+          "type": "string",
+          "description": "Login user"
+        },
+        "password": {
+          "type": "string",
+          "description": "Password for SSH transport authentication"
+        },
+        "private-key-content": {
+          "type": "string",
+          "description": "Contents of private key for SSH"
+        },
+        "port": {
+          "type": "integer",
```

Comment:
I think this can be `["integer", "string"]` like for winrm

Reply:
It seems better to be strict. Maybe we should change winrm to just integer?

Reply:
I agree that strict is better. Added some logic for the winrm transport to get it through appveyor. Could add that same checking to expected integer fields in ssh as well.